src/utils/index.js

```diff
@@ -227,6 +227,18 @@ export async function fetchPackageCompatibilities() {
   return request;
 }
 
+export async function retrieveHostingPlatformData(inputValues) {
```

Comment:
As this method is used to deploy (POST), can we rename this to publishViz or deployViz ?

Reply:
hey yeah good suggestion, i couldn't think of any good name for it, thank you